src/Fantomas/Program.fs

```diff
@@ -16,6 +18,7 @@ type Arguments =
     | [<Unique>] Out of string
     | [<Unique>] Check
     | [<Unique>] Daemon
+    | [<Unique>] Parallel
```

Comment:
I would introduce a `Sequential` flag instead and use parallel by default.

Reply:
✔️